.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,18 +12,20 @@
  - `chore`: all chores/tests will be excluded from the CHANGELOG
  -->
 
-## Problem
```

Comment:
I think we can make the whole template a bit smaller. We probably just need JIRA Ticket, GitHub issue, [Breaking], description.